inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/RowDataDebeziumDeserializeSchema.java

```diff
@@ -126,7 +131,17 @@ public static Builder newBuilder() {
         this.resultTypeInfo = checkNotNull(resultTypeInfo);
         this.validator = checkNotNull(validator);
         this.changelogMode = checkNotNull(changelogMode);
-        this.sourceMetricData = sourceMetricData;
+        this.metricOption = metricOption;
+    }
+
+    @Override
+    public void open() {
+        if (metricOption != null) {
+            LOG.info("SourceMetricData create -----------");
```

Comment:
please keep log format consistent with other places

Reply:
fix it，thx～